cpanfile.snapshot

```diff
@@ -3839,6 +3839,19 @@ DISTRIBUTIONS
       IO::WrapTie::Slave 2.110
     requirements:
       ExtUtils::MakeMaker 0
+  IPC-Run-0.94
```

Comment:
I think this was from Archive::Tar::Wrapper. I use sublime as my editor and  when I switch branches cpanfile.snapshot keeps what I have installed before if I don't run carton install again.